<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messages</class>
 <widget class="QWidget" name="messages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <property name="styleSheet">
   <string notr="true">messages
{
	background-color:#0a0612;
}
QPushButton
{
	background-color:#8a02b2;
	color:#1f1137;
	border-radius:20px;
}
QPushButton:hover
{
	background-color:#341d5c;
	color: #ffffff;
}
QLabel
{
	color:#ad03de;
}
QLineEdit
{
	background-color:#673ab7;
	color:#ffffff;
	border-radius:16px;
}
QLineEdit:hover
{
	background-color:#5d34a5;
}
QPlainTextEdit
{
	color:#ffffff;
	background-color:#3e236e;
	border-radius:12px;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>831</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPlainTextEdit" name="pte_message6">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="pte_message1">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPlainTextEdit" name="pte_message5">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="pte_message4">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pte_message2">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="pte_message3">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_accept1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_accept2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_accept3">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept3">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_accept4">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept4">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_accept5">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept5">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="btn_accept6">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_notaccept6">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Not Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>70</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>630</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QPushButton" name="btn_up">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_down">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
